### ISSUE TITLE
Non systemd support

### ISSUE THREAD

**Issue by RupankarGhosh** [netdata/netdata#8766]
Is it possible to start netdata on those systems which doesn't uses 
```
systemd
```

**Comment by thiagoftsm** [comment]
Hi @RupankarGhosh ,

It is completely possible, I always ran Netdata on `Slackware current`, in my case I call it direct on terminal ( `netdata` ), but case you want Netdata always starting, it is only necessary to add an entry in your `rc.local` on Slackware it is present at `/etc/rc.d/`.

Best regards!

**Comment by RupankarGhosh** [comment]
Thank you for your fast reply. I'm wandering that is it possible to run netdata on android. I'm using debian on android without systemd. I'm only getting this problem "access to file is not permitted: /usr/share/netdata/web/index.html"

**Comment by thiagoftsm** [comment]
I never installed Netdata on Android, but in normal situation when we receive this error message from a Netdata running on a server, this means that Netdata is running with an user that does not have `read` permission on directory `usr/share/netdata/web/`. 

How did you install Netdata?

**Comment by ilyam8** [comment]
> "access to file is not permitted: /usr/share/netdata/web/index.html"

https://github.com/netdata/netdata/issues/8015

**Comment by RupankarGhosh** [comment]
@thiagoftsm simply I installed with 
apt-get install netdata. As it was on default package repository.

**Comment by thiagoftsm** [comment]
Thank you @RupankarGhosh ,

I assigned our SRE team for they take a look in the problem you are having.

**Comment by prologic** [comment]
Yes this is something we want to do and should support. We _should_ have support for htis today as there are non-systemD SysV init scripts in the source tree. Please have a look there to see if those can work for your system.

I've added this to our backlog and ask that the community please put up a PR that improves this for their non-systemD systems. I have only a couple myself (CRUX and uLInux) that are non-systemD myself but I don't currently have the bandwidth to get to this just yet. So PRs welcome!

**Comment by iluvatar1** [comment]
> Hi @RupankarGhosh ,
> 
> It is completely possible, I always ran Netdata on `Slackware current`, in my case I call it direct on terminal ( `netdata` ), but case you want Netdata always starting, it is only necessary to add an entry in your `rc.local` on Slackware it is present at `/etc/rc.d/`.
> 
> Best regards!

Hi @thiagoftsm, (please forgive me, I do not want to hijack the thread) 
would you mind pointing me out on how you install netdata on slackware-current? Currently the slackbuild from slackbuilds.org creates a version that cannot be used to sync to netdatacloud, and installing mosquitto and libwebsockets from slackbuilds is "useless" since netdata looks for a static lib version. For now I had to install netdata using the kickstart script and forget about creating a package for easier maintenance.
Thanks in advance.

**Comment by thiagoftsm** [comment]
Hello @iluvatar1 ,

Of course I can help you, on Slackware current I install running the following commands:

```
$ git clone https://github.com/netdata/netdata.git
$ cd netdata
$ su
# ./netdata-installer.sh --disable-lto --dont-wait --dont-start-it
```

After this I edit `/etc/rc.d/rc.local` and I add  the following line:

```
/usr/sbin/netdata 
```

This will enable the majority of Netdata default plugins, the `eBPF` plugin won't be enable, because it is necessary to recompile the kernel to enable `kprobes`, if you want to know the steps to recompile kernel and what is necessary to activate, please, let me know.

**Comment by Ferroin** [comment]
@iluvatar1 Of possible note, once #10315 is merged and the code finally ends up in the version provided by the Slackware packages, Cloud connectivity should just work without any special effort from either Slackware or any users. I don’t currently know for certain when that may be merged though.

**Comment by iluvatar1** [comment]
Hi @thiagoftsm and @Ferroin , 
thanks for the info. On the last weekend, I was "playing"  with the netdata slackbuild, and at the end I was able to use it to create a netdata that is cloud aware. The basic problem is that netdata wants mosquitto and libws to come from the static included libs, so the slackbuilds from those packages do not work/are not detected. After many tries I came up with the following simple modification to the slackbuild:

- Add  the following two lines just before the configure line:
`
 bash packaging/bundle-mosquitto.sh ./
 bash packaging/bundle-lws.sh ./
`
 This builds the static dependencies that netdata needs to be cloud aware.
- Add the following last line to the configure command 
`
    --with-bundled-lws=./externaldeps/libwebsockets/
`
With this I can use the slackbuild to successfully build a netdata package that can also connect to the netdata cloud (tested). The advantage is not only to have a packaged that can be upgraded/removed ..., but also to use the /etc/rc.d/rc.netdata script that come with the slackbuild.

I hope that the referenced change by @Ferroin will remove the need for this modifications.

Have a nice day.
